data/Paul_1874_Iwein/metadata.txt

```diff
@@ -0,0 +1,26 @@
+publicationType : "article"
+publicationTitle : "Über das gegenseitige Verhältnis der Handschriften von Hartmanns Iwein"
+publicationDate : "1874"
+publicationPlace : ""
+publicationSeries : "Beiträge zur Geschichte der deutschen Sprache und Literatur"
+publicationNum: "1"
+publicationStemmaNum: "1"
+publicationAuthors : "Paul, Hermann"
+publicationPage : "288-401"
```

Comment:
include the page where the stemma is found ? (that's the purpose I originally thought for this field).

Reply:
Done